### DISCUSSION TITLE
Only 1 monitor can have brightness changed, the other can't

### DISCUSSION THREAD

**Discussion by m1strust** [xanderfrangos/twinkle-tray#861] [Monitor Compatibility Issues]
I have 2 Sceptre M25 monitors, both connected with the same DisplayPort cable and only my main monitor can have the brightness changed. Both are detected and I can slide the brightness bar for the secondary monitor but it doesn't change anything.

**Comment by xanderfrangos** [comment]
When you say it's the same cable, do you mean it's the same model of cable, or that you're daisy-chaining the monitors into a single DisplayPort port? Usually when something like this happens, switching around which ports are used can help. Also, which version of Twinkle Tray are you on?

**Reply by m1strust** [reply]
2 seperate DP cables plugged into 2 different ports. Just tried switching them around to no avail. Just updated to 1.16.3 and restarted and the problem persists

**Reply by m1strust** [reply]
DP cables are the exact same I bought them in a 2 pack

**Reply by m1strust** [reply]
Nevermind, problem solved. Went to picture settings -> preset and set it to user and it worked. Appreciate the response

**Reply by ricod3** [reply]
hi @m1strust,

what do you mean with 'picture settings'? I have the same issue but when opening picture settings, i don't see anywhere the 'preset'-option you mentioned. Am I blind?:

![grafik](https://github.com/user-attachments/assets/568f7500-a6d7-4059-952a-7e7e9f2fc4af)

Can you show in a screenshot what you mean?

Thanks

**Reply by m1strust** [reply]
I opened the monitor settings through a physical button on the back.

on my monitor ;
- Monitor settings (button on back)
- Picture
- change Preset to User

Your settings may be phrased differently but just look for something to make your monitor settings custom, not on any preset.

**Reply by ricod3** [reply]
Thanks, it worked.
